app/jobs/google/search_keyword_job.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Google
+  class SearchKeywordJob < ApplicationJob
+    queue_as :default
+
+    def perform(keyword_id)
+      keyword = Keyword.find(keyword_id)
+      search_result = Google::SearchService.new(keyword.name).call
+      update_keyword(keyword, search_result)
+    end
+
+    private
+
+    def update_keyword(keyword, search_result)
+      if search_result
+        keyword.update search_result.merge(status: 1)
```

Comment:
Let use enum instead, we try to avoid the magic number / magic string by reference to the model constant or enum.
Please update other place as well

```suggestion
        keyword.update search_result.merge(status: Keyword.status[:parsed)
```

Reply:
Small additional note:

if you use string enum on the model: `enum status: { in_progress: 'in_progress', parsed: 'parsed', failed: 'failed' }` then you can have a shorter yet explicit query like:

```suggestion
        keyword.update search_result.merge(status: 'parsed')
```

In terms of performance, it is true that using a small int is better than using a string, but it remains a negligible difference in the context of this application.

Reply:
Updated to string.